dot-net-sdk/validators/BanditEvaluator.cs

```diff
@@ -105,7 +105,15 @@ public BanditEvaluator(int totalShards = 10_000)
     {
         var numberOfActions = actionScores.Count;
 
-        var bestAction = actionScores.Aggregate((currentmax, next) => currentmax.Value > next.Value ? currentmax : next);
+        // var bestAction = actionScores.Aggregate((currentmax, next) =>
+        // {
+        //     if (currentmax.Value == next.Value) {
+        //         return currentmax.Key.CompareTo(next.Key) < 0 ? currentmax : next;
+        //     }
+        //     return currentmax.Value > next.Value ? currentmax : next;
+        // });
```

Comment:
I'm fine axing this aggregate biasing towards the readability of below. Practically, I don't imagine it making much difference.